conda.recipe/build.sh

```diff
@@ -0,0 +1,6 @@
+SRC_DIR=$RECIPE_DIR/..
+cd $SRC_DIR
+npm install .
+npm run build
+python setup.py install
+python -m nbpresent.install --force --enable
```

Comment:
Is there a handy way to do runtime flags? Checking the doc...

On Tue, Nov 10, 2015 at 9:13 AM Damián Avila <notifications@github.com>
wrote:

> In conda.recipe/build.sh
> <https://github.com/ContinuumIO/nbpresent/pull/7#discussion_r44409768>:
>
> > @@ -0,0 +1,6 @@
> > +SRC_DIR=$RECIPE_DIR/..
> > +cd $SRC_DIR
> > +npm install .
> > +npm run build
> > +python setup.py install
> > +python -m nbpresent.install --force --enable
>
> @damianavila <https://github.com/damianavila> in your experience, where
> do most people install extensions (i.e. global/user)?
>
> Usually in user space...
>
> Is there an expectation that the global /share location is really shared
> by every user/app?
>
> In some deployment like JupyterHub, maybe it makes sense to have a
> system-wide installation...
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/ContinuumIO/nbpresent/pull/7/files#r44409768>.
>